generator/src/main/java/com/scottlogic/deg/generator/generation/string/RegexStringGenerator.java

```diff
@@ -216,7 +216,9 @@ private String buildStringFromNode(Node node, int indexOrder) {
             passedStringNbrInChildNode = passedStringNbr;
         }
         for (Node childN : node.getNextNodes()) {
-            passedStringNbrInChildNode += childN.getMatchedStringIdx();
+            // TODO AF delete this commented out old code before merge:
```

Comment:
@afroggattsl does this comment need addressing

Reply:
i will remove these before merge.  there's a few and as long as they have TODO by them its fine as i will fix them

Reply:
in fact i'll do it now

Reply:
done